src/main/java/com/spotify/github/v3/clients/GithubAppClient.java

```diff
@@ -69,8 +77,18 @@ public CompletableFuture<List<Installation>> getInstallations() {
    * @return a list of Installation
    */
   public CompletableFuture<Installation> getInstallation() {
+    return maybeRepo.map(repo-> github.request(
+        String.format(GET_INSTALLATION_REPO_URL, owner, repo), Installation.class, extraHeaders)).orElseGet(() -> getOrgInstallation(owner));
+  }
+
+  /**
+   * Get an installation of an org
+   * @param owner the owner/organisation
+   * @return an Installation
+   */
+  private CompletableFuture<Installation> getOrgInstallation(final String owner) {
```

Comment:
Why not use `this.owner` instead of an additional Param here?

Reply:
+1